front_end/webserver.py

```diff
@@ -960,6 +960,7 @@ def post(self, course, assignment, exercise):
             exercise_details["show_answer"] = self.get_body_argument("show_answer") == "Yes"
             exercise_details["show_student_submissions"] = self.get_body_argument("show_student_submissions") == "Yes"
             exercise_details["enable_pair_programming"] = self.get_body_argument("enable_pair_programming") == "Yes"
+            exercise_details["check_code"] = self.get_body_argument("check_code_text").strip().replace("\r", "")
```

Comment:
We're stripping white space here, so we should not need to strip it again in helper.py function. It's a minor thing, but every little bit helps in terms of simplifying the code and speeding things up.

The same is true for test_code.